python/Gaffer/Application.py

```diff
@@ -86,6 +94,10 @@ def root( self ) :
 	## Called to run the application and return a status value.
 	def run( self ) :
 
+		if self.parameters()["help"].getTypedValue() :
+			self.__formatHelp()
+			return
```

Comment:
Sure. Should it be returning `IntData` or an `int`?

PS. It seems some apps (like stats) are not following that rule right now. `run()` returns `self.__run()` returns `self._run()` which for `stats` returns `None`